static/css/components/header-bar.less

```diff
@@ -538,6 +538,7 @@
     .logo-component {
       .logo-icon {
```

Comment:
Could you add a comment here that these should be kept in sync with the width and height attributes of logo-icon ?